apps/api/src/database/schema/syncing/defs.ts

```diff
@@ -65,4 +65,4 @@ export const objectIndex = pgTable(
       name: "object_index_agent_id_fkey",
     }).onDelete("cascade"),
   ],
-);
```

Comment:
This was a result of running `pnpm format` I'm not sure how it made it to main without end of file return

Reply:
hm odd. there are some chore-related repo things we need to look at...e.g., `format` is a root-level command, and it isnt enforced in CI (or pre-commit hooks, if we want to get more restrictive).